presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -2,7 +2,20 @@
 JDBC Driver
 ===========
 
-Presto can be accessed from Java using the JDBC driver.
+The Presto JDBC (Java Database Connectivity) driver allows users to access 
```

Comment:
I would assume people commonly use "JDBC driver" without expanding the acronym, so expanding it here can make the text less clear, not more.
What about we make "JDBC driver" a link to a webpage explaning what JDBC is?

Reply:
Great idea. Almost every decent page is either an obscure blog (ie, not really reference material), or talks about someone else's driver in particular. Are we ok with wikipedia links in general? https://en.wikipedia.org/wiki/JDBC_driver

Reply:
I am fine. https://docs.oracle.com/javase/tutorial/jdbc/overview/index.html may be useful too

Reply:
Both are okay.